libs/interpret-vision/src/lib/VisionExplanationDashboard/Controls/Flyout.tsx

```diff
@@ -0,0 +1,168 @@
+// Licensed under the MIT License.
+
+import {
+  Icon,
+  Image,
+  ImageFit,
+  List,
+  Panel,
+  PanelType,
+  FocusZone,
+  Stack,
+  Text
+} from "@fluentui/react";
+import { localization } from "@responsible-ai/localization";
+import React from "react";
+
+import { IDatasetSummary } from "../Interfaces/IExplanationDashboardProps";
+
+import { flyoutStyles } from "./Flyout.styles";
+import { IListItem } from "./ImageList";
+
+export interface IFlyoutProps {
+  data: IDatasetSummary;
+  isOpen: boolean;
+  item: IListItem | undefined;
+  callback: () => void;
+}
+
+export class IFlyoutState {}
+
+export class Flyout extends React.Component<IFlyoutProps, IFlyoutState> {
+  public constructor(props: IFlyoutProps) {
+    super(props);
+    this.state = {};
+  }
+
+  public render(): React.ReactNode {
+    const classNames = flyoutStyles();
+
+    const item = this.props.item;
+
+    const list = [];
+
+    for (let i = 0; i < 30; i++) {
+      list.push({ title: "Feature 1", value: "value" });
+    }
+
+    return (
+      <FocusZone>
+        <Panel
+          headerText={localization.InterpretVision.Dashboard.panelTitle}
+          isOpen={this.props.isOpen}
+          closeButtonAriaLabel="Close"
+          onDismiss={this.props.callback}
+          isLightDismiss
+          type={PanelType.custom}
+          customWidth="700px"
+          className={classNames.mainContainer}
+        >
+          <Stack tokens={{ childrenGap: "l1" }}>
+            <Stack.Item>
+              <div className={classNames.line} />
+            </Stack.Item>
+            <Stack.Item>
+              <Stack
+                horizontal
+                tokens={{ childrenGap: "l1" }}
+                horizontalAlign="space-around"
+                verticalAlign="center"
+              >
+                <Stack.Item>
+                  <Stack
+                    horizontal
+                    tokens={{ childrenGap: "s1" }}
+                    horizontalAlign="center"
+                    verticalAlign="center"
+                  >
+                    <Stack.Item className={classNames.iconContainer}>
+                      <Icon
+                        iconName={
+                          item?.predictedY !== item?.trueY
+                            ? "Cancel"
+                            : "Checkmark"
+                        }
+                        className={
+                          item?.predictedY !== item?.trueY
+                            ? classNames.errorIcon
+                            : classNames.successIcon
+                        }
+                      />
+                    </Stack.Item>
+                    <Stack.Item>
+                      <Text variant="large" className={classNames.title}>
+                        {item?.predictedY !== item?.trueY
+                          ? localization.InterpretVision.Dashboard.titleBarError
+                          : localization.InterpretVision.Dashboard
```

Comment:
I figured that since this text (i.e. "Error Instances") appears in both the dashboard (in the title bar for the image list) and the flyout, I could just reuse it. Should I refactor anyways?